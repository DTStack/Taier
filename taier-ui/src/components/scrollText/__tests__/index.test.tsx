import { render } from '@testing-library/react';
import ScrollText from '..';

describe('Test ScrollText Component', () => {
    it('Should match snapshot', () => {
        expect(render(<ScrollText />).asFragment()).toMatchSnapshot();
    });
});
