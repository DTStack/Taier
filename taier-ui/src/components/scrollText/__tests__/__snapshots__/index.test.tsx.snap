// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ScrollText Component Should match snapshot 1`] = `
<DocumentFragment>
  <input
    class="cell-input"
    data-testid="test-scroll-text"
    readonly=""
    style="height: 28px; margin: 5px 5px 5px 0px; width: 100%; text-align: left; background-color: transparent; background-image: none;"
    value=""
  />
</DocumentFragment>
`;
